Pipfile

```diff
@@ -58,6 +58,7 @@ trino = "*"
 unleashclient = "<5"  # new version breaks for unknown reason
 watchtower = "==1.0.6"  # new version breaks for unknown reason
 whitenoise = ">=5.0"
+pydantic = "*"
```

Comment:
We should required Pydantic >=2 since there are [breaking changes](https://docs.pydantic.dev/2.0/migration/) from v2.